support/contracts.md

```diff
@@ -0,0 +1,84 @@
+# Support Contracts
+
+In addition to [community](/community) support,
+[OpenSSL Software Services](/community/contacts.html) offers three different
+types of support contract. If you have specific requirements not addressed
+by any of these plans, or for more information, please contact us at
+<osf-contact@openssl.org> to discuss custom arrangements.
+
+Please see the [list of definitions](#definitions) at the bottom of the page
+for the definitions used below.
+
+[Premium](#premium) Enterprise Level Support
+:   Designed for the large enterprise utilising OpenSSL extensively in
+    product lines or critical infrastructure.
```

Comment:
This is a definition list entry in markdown as shown in https://www.markdownguide.org/extended-syntax/#definition-lists
Unfortunately, GFM doesn't know this, but pandoc does.